src/app/globals.css

```diff
@@ -80,29 +79,40 @@
   }
 }
 
-.rr-player {
-  box-shadow: none !important;
-  @apply border border-gray-200 rounded-sm
-}
-
-.rr-controller {
-  @apply bg-transparent !important;
+.switch input[type='checkbox']:checked+label:before {
+  background: var(--player-primary) !important;
 }
 
-.rr-controller .active, .rr-progress__handler  {
+.switch input[type='checkbox']+label:before {
   background: var(--player-primary) !important;
+  opacity: 0.4 !important;
 }
 
-.rr-progress__step {
-  background: var(--player-primary) !important;
-  opacity: 0.2 !important;
+/* styles.css */
+.CollapsibleContent {
+	overflow: hidden;
+}
+.CollapsibleContent[data-state="open"] {
+	animation: slideDown 100ms ease-out;
+}
+.CollapsibleContent[data-state="closed"] {
+	animation: slideUp 100ms ease-out;
 }
 
-.switch input[type='checkbox']:checked+label:before {
-  background: var(--player-primary) !important;
+@keyframes slideDown {
+	from {
+		height: 0;
+	}
+	to {
+		height: var(--radix-collapsible-content-height);
+	}
```

Comment:
Nit:

Wow, what indentation do you use here? It seems inconsistend with [this line](https://github.com/metabase/metabase-debugger/pull/7/files#diff-4f206759c961f544a48464ad2ee231adc87e72593d03a0004b68c17771f7a412R83) for example.

Reply:
good eyes! 👍  I copy-pasted this from the radix docs on the collapsible component, because I wanted to add animation

Reply:
RadixUI <3